inst/sensorreading/sensorreading.Rmd

```diff
@@ -63,10 +63,14 @@ if(!identical(tbl, "The dataset requested is empty.")){
 
 ```{r,echo=FALSE,result='asis',comment=NA,warning=FALSE}
 if(!identical(tbl, "The dataset requested is empty.")){
-  tbl2 <- srsQualifiersTable(data, tbl)
-  if(!is.null(tbl2) && nrow(tbl2) > 0){
+  quals <- as.data.frame(data$readings$qualifiers)
+  if(!is.null(quals) && length(quals) > 0){ 
+    quals <- quals[, c("code", "identifier", "displayName")]
+    tbl2 <- formatQualifiersTable(quals)
     if(!is.null(tbl2) && nrow(tbl2) > 0){
-      kable(tbl2, align='l', row.names=FALSE, caption = "Qualifiers")
+      if(!is.null(tbl2) && nrow(tbl2) > 0){
```

Comment:
I guess so, I left it here from before, but I can try removing it and see if it matters or not. I find it hard to work with the Rmd files because I haven't figured out how really to debug within them